schedule/migration/tls_389ds_sssd.yaml

```diff
@@ -0,0 +1,31 @@
+name: tls_389ds_sssd
+description:    >
+    This is for 389ds and sssd authentication before and after migration check
+vars:
+  ORIGIN_SYSTEM_VERSION: '%HDDVERSION%'
```

Comment:
yamllint --strict schedule/migration/tls_389ds_sssd.yaml
I remembered that there should be 4 blank space here :)